### PR TITLE
fix: align with forked from

### PR DESCRIPTION
**Pull request by VadimDor**

### CODE REVIEW

**Comment by VadimDor** [comment]
lets merge it

File: .github/dependabot.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 2
 updates:
   # dependabot for GitHub Actions for this repo
   - package-ecosystem: "github-actions"
-    directory: "./github"
```
Comment (radiocab) [inline .github/dependabot.yml:5]: aprove changes

File: template/.github/dependabot.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 version: 2
 updates:
   - package-ecosystem: "github-actions"
-    directory: "./github"
```
Comment (radiocab) [inline template/.github/dependabot.yml:4]: approve

**Review by radiocab — CHANGES_REQUESTED** [review]
aprove all

**Review by radiocab — APPROVED** [review]

**Pull request closed without merging**